framework/src/media/audio/audio_manager.c

```diff
@@ -123,15 +130,35 @@ audio_manager_result_t find_input_audio_card(void)
 			|| (dir_entry->d_name[1] != 'c')
 			|| (dir_entry->d_name[2] != 'm')
 			|| (sscanf(&dir_entry->d_name[3], "C%uD%u%c", &card_id, &device_id, &type) != 3)) {
-			closedir(dir_info);
-			return AUDIO_MANAGER_INVALID_DEVICE_NAME;
+			continue;
 		}
 
 		if ((type == 'c') && (card_id < CONFIG_AUDIO_MAX_INPUT_CARD_NUM) && (device_id < MAX_AUDIO_DEVICE_NUM)) {
+			struct audio_caps_desc_s caps_desc;
```

Comment:
CONFIG_AUDIO_EXCLUDE_VOLUME should be considered.

Reply:
'#ifndef CONFIG_AUDIO_EXCLUDE_VOLUME' is added.